packages/quantic/force-app/main/default/lwc/quanticSummary/__tests__/quanticSummary.test.js

```diff
@@ -0,0 +1,124 @@
+jest.mock('c/quanticHeadlessLoader');
+import QuanticSummary from 'c/quanticSummary';
+import {buildCreateTestComponent, cleanup, flushPromises} from 'c/testUtils';
+import * as quanticHeadlessLoader from 'c/quanticHeadlessLoader';
+
+const headlessLoaderMock = jest.mocked(quanticHeadlessLoader);
+const engineMock = {
+  id: 'mockEngine',
+  dispatch: jest.fn(),
+};
+
+let updateState;
+const summaryControllerMock = {
+  subscribe: jest.fn((listener) => {
+    updateState = async () => {
+      listener();
+      await flushPromises();
+    };
+    return () => {}; // Fake unsubscribe function
```

Comment:
just for consistency, we usually mock the unsubscribe like this:
<img width="318" alt="image" src="https://github.com/user-attachments/assets/8a474df5-38d7-4b92-a596-918b162e3e33" />

Reply:
C'est bizzare messemble, la je save la callback method dans une variable pour la caller a chaque test, mais je ne valide pas vraiment que unsubscribe est callé. Dans ton exemple tu check aussi que c'est callé?

Reply:
yup tu peux search `unsubscriber` et ca va te montrer des examples